### PR TITLE
Add input box to chronos-ui for filtering table rows by job name, state, or status

### PR DESCRIPTION
**Pull request by samirrayani**
next step: build this fork (nextbigsoundinc/chronos) via Jenkins and deploy to [chronos-main-prod.sv3.tnbsound.com/](http://chronos-main-prod.sv3.tnbsound.com/#/)

screen shot 2017-09-29 at 10 58 38 pm

/cc @nextbigsoundinc/systems

### CODE REVIEW

**Review by cynthiameng — APPROVED** [review]

**Comment by bfemiano** [comment]
Fuck. Yes.

**Review by mpetrovich — APPROVED** [review]
😍

File: src/main/resources/ui/components/Main.js [hunk 1]
```diff
@@ -5,6 +5,16 @@ import {observer} from 'mobx-react'
 
 @observer
 export default class Main extends React.Component {
+  constructor(props) {
+    super(props);
```
Comment (mpetrovich) [inline src/main/resources/ui/components/Main.js]: Semicolon here. It looks like the style of the source is to omit semicolons.

**Comment by choshun** [comment]
Nice! Had no clue we had a react app.

**Comment by samirrayani** [comment]
@choshun we didn't until we forked this from mesos :)

**Pull request merged**